lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -92,5 +90,12 @@ private function build($data) {
     $this->created_at = $data->timestamp;
     $this->campaign = Campaign::get($data->nid);
     $this->campaign_run = $data->run_nid;
+
+    try {
+      $this->rbid = Reportback::get($data->rbid);
```

Comment:
one quickie, forgot we should probs name this property `$this->reportback = ...` and not `rbid` since it'll return a reportback object, not just the rbid